composer.json

```diff
@@ -44,6 +44,8 @@
         "mockery/mockery": "1.0.*@dev",
         "fzaninotto/faker": "1.2.*",
         "codeclimate/php-test-reporter": "dev-master",
-        "johnkary/phpunit-speedtrap": "~1.0@dev"
+        "johnkary/phpunit-speedtrap": "~1.0@dev",
+        "fabpot/php-cs-fixer": "^1.10",
+        "phpmd/phpmd": "^2.2"
```

Comment:
@yitznewton 

From what I heard and read, the `^` operator works best for packages which you trust to have embraced semantic versioning, as for example

```
$ composer require --dev fabpot/php-cs-fixer:^1.9.3`
```

will always resolve to `<=2.0.0` and `>=1.9.3`, while

```
$ composer require --dev fabpot/php-cs-fixer:~1.9.3
```

will resolve to anything `>=1.9.3` and `<1.10` (if I understood it correctly).